index.js

```diff
@@ -50,11 +53,12 @@ function _byteLength (b64, validLen, placeHoldersLen) {
   return ((validLen + placeHoldersLen) * 3 / 4) - placeHoldersLen
 }
 
-function toByteArray (b64) {
+function toByteArray (b64, alphabet) {
```

Comment:
Why does decoding need to take an alphabet? You set `revLookups.url = revLookups._` anyway, so this seems like it adds nothing. Am I missing something?

Reply:
i felt like keeping the external api symmetrical was valuable, i think i prefer it this way but happy to change if you like.

Reply:
I see. I agree that symmetry is nice, but I think that's outweighed by having a parameter that has no effect. Can you change it?

Reply:
ok. how much do you want to remove? the entire revlookups table? as in, go back to having a single variable? if anyone wants to experiment with or add additional alphabets later it will be much more difficult...

Reply:
basically, now that i'm looking at it, i can see it's not just the interface, it's the implementation as well. the parameter doesn't have "no effect", it's just that the default alphabet happens to include a hack (for node compatibility) to allow decoding url safe base 64, so we're reusing the default reverse lookup for both alphabets to save a little memory.

so yea we could remove just the parameter from the interface, but then why bother keeping the symmetrical implementation?

Reply:
it also seems like it might be worth at least encouraging user code to be specific about what is being decoded if possible - i'm thinking that reading `b64.toByteArray(str, 'url')` in my source might save me some confusion down the road

Reply:
and just to really beat it to death... from https://tools.ietf.org/html/rfc4648#section-5:

```
This encoding may be referred to as "base64url".  This encoding
should not be regarded as the same as the "base64" encoding and
should not be referred to as only "base64".  Unless clarified
otherwise, "base64" refers to the base 64 in the previous section.
```